java/server/src/org/openqa/selenium/remote/server/rest/ResultConfig.java

```diff
@@ -166,6 +168,34 @@ public Response handle(Command command) throws Exception {
     return response;
   }
 
+  private String removeCredentials(String command) {
+    if (Strings.isNullOrEmpty(command)) {
+      return command;
+    }
+
+    if (command.contains("send keys")) {
+      java.util.regex.Pattern credentialsPattern =
+        java.util.regex.Pattern
+          .compile("(.*)(send keys:).*(])", java.util.regex.Pattern.CASE_INSENSITIVE);
```

Comment:
Compile the regex once by moving it as a static final in the class

Reply:
This regex is a bit confusing. Can you tell me how this would ideally show up in the logs before redaction?

Reply:
Made it a static final for the class 

Also added some sample logs to the bottom of the file